ui/modal/modalCollectionCreate/internal/collectionCreate/index.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import CollectionCreate from './view';
+import { selectPermanentUrlForUri } from 'redux/selectors/claims';
+import { doToast } from 'redux/actions/notifications';
+
+const select = (state, props) => {
+  const { uri } = props;
+
+  return {
+    // lists can only use permanent_url
+    uri: selectPermanentUrlForUri(state, uri),
+  };
```

Comment:
- [ ] Is `uri` needed when creating private collection? It is also passed in from the calling side, so should clean up there as well if not needed.